spec/bin/mcb_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
```

Comment:
Is there a point with `rails_helper`?

Just thinking its a console app no longer a rails app

Reply:
Good spot.

Some commands _will_ load the Rails environment, though, but I guess we can worry about those later.